src/TensorFlowNET.Keras/Engine/Model.Fit.cs

```diff
@@ -193,6 +215,123 @@ public partial class Model
                     {
                         logs["val_" + log.Key] = log.Value;
                     }
+                    callbacks.on_train_batch_end(End_step, logs);
```

Comment:
Because some new log has been added to the logs after the evaluation, so we need to call it again